docker_images/fasttext/tests/test_api_language_identification.py

```diff
@@ -0,0 +1,96 @@
+import json
```

Comment:
This file is not needed, and instead the text classification test should have an additional test for language identification